sickbeard/providers/newznab.py

```diff
@@ -251,6 +254,8 @@ def _make_provider(config):
 
             if len(values) == 9:
                 name, url, key, category_ids, enabled, search_mode, search_fallback, enable_daily, enable_backlog = values
+            elif len(values) == 10:
```

Comment:
This would work, an alternative is this:
```python

@staticmethod
def _make_provider(config):
    if not config:
        return None

    try:
        values = config.split('|')
        # Pad values with None for each missing value
        values.extend([None for x in range(len(values), 10)])

        (name, url, key, category_ids, enabled,
         search_mode, search_fallback, enable_daily, enable_backlog,
         enable_manualsearch) = values

    except ValueError:
        logger.log('Skipping Newznab provider string: {config!r}, incorrect format'.format
                   (config=config), logger.ERROR)
        return None

    new_provider = NewznabProvider(
        name, url, key=key, catIDs=category_ids,
        search_mode=search_mode or 'eponly',
        search_fallback=search_fallback or 0,
        enable_daily=enable_daily or 0,
        enable_backlog=enable_backlog or 0,
        enable_manualsearch=enable_manualsearch or 0)
    new_provider.enabled = enabled == '1'

    return new_provider
```